v2.5.0 version bump & highlights (#13738)
WalletWasabi/WalletWasabi: WalletWasabi/Announcements/ReleaseHighlights.md

## Release Highlights
#### 💪 3rd Party Providers for Fee & Exchange Rate<br/>
#### 👨‍🔧 Quality of Life Features<br/>
#### 📦 Backend and Coordinator packaged for Linux<br/>

## Release Summary
Wasabi Wallet v2.5.0 continues to enhance Wasabi's resilience and improve the user interface to reduce frustration.

### 💪 3rd Party Providers for Fee & Exchange Rate

Fee rate estimations and exchange rate providers are now configurable.

> Previously, this information was provided by Wasabi's backend servers. This change aligns with our long-term goal of completely removing the backend.

Reviewer: ```suggestion
Previously, this information was provided by Wasabi's backend server. This change aligns with our long-term goal of completely removing dependence on the backend.
```

Reviewer: or maybe remove `server` completely

After this release, Wasabi will be significantly closer to achieving that goal, with most basic wallet features functioning without an active backend connection. You can read more about the remaining steps [here](https://github.com/orgs/WalletWasabi/discussions/13661).

### 👨‍🔧 Quality of Life Features

In the last version, we introduced a donation button to gauge community interest in funding Wasabi's future. We have been overwhelmed by the support and interest shown by our community, including long-time users and privacy enthusiasts.

We want to thank you for believing in Wasabi and for your support, whether through using the software, providing feedback, contributing financially, or contributing to the open-source project in any form.

To show our gratitude, we've designed this release for you, our community. We've finally addressed some of the oldest bugs and feature requests that have been long-awaited!

Listing all the improvements would be too lengthy, but here are some examples:

> - More & better-designed settings (specifically for Coinjoin)

Reviewer: ```suggestion
- More & better-designed settings (specifically for coinjoin)
```

Reviewer: nit

> - Randomly skipping rounds was disabled

Reviewer: ```suggestion
- Randomly skipping rounds was removed
```

- Change avoidance suggestions are now more precise

We expect these improvements to significantly enhance the overall experience of using Wasabi as your daily Bitcoin wallet.

### 📦 Backend and Coordinator packaged for Linux

The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). This update simplifies the deployment process for Linux users, making it straightforward to set up their backend servers and coordinators.

> This ease of use lowers the barrier to contributing to the resiliency of Wasabi's infrastructure against potential attacks and technical failures. It also makes it possible to create private Wasabi infrastructures with private coordinators and backends.

Reviewer: > This ease of use lowers the barrier to contributing to the resiliency of Wasabi's infrastructure against potential attacks and technical failures

maybe simplify with something like: makes it easier to run a backend and/or coordinator

> It also makes it possible to create private Wasabi infrastructures with private coordinators and backends.

this is not so clear imo, what do you mean to say

Reviewer 2: > what do you mean to say

I mean that you can run a backend/coordinator for your community, company, club or whatever. Imagine your company hosts a coordinator and all employees can access to the vlan and schedule payments in coinjoins.

## Release Highlights

> #### 💪 3rd Party Providers for Fee & Exchange Rate<br/>

Reviewer: ```suggestion
#### 💪 3rd Party Providers for Fee Estimation & Exchange Rate<br/>
```

Author: 1 word less you won

> #### 💪 3rd Party Providers for Fee & Exchange Rate<br/>

Author: NACK, these titles must be short

#### 👨‍🔧 Quality of Life Features<br/>
#### 📦 Backend and Coordinator packaged for Linux<br/>

## Release Summary

> Wasabi Wallet v2.5.0 continues to enhance Wasabi's resilience and improve the user interface to reduce frustration.

Reviewer: ` improve the user interface to reduce frustration.`
this is subjective, I'd rephrase/remove

Author: I dislike using this kind of acronyms UI/UX because many users won't know what they mean. I pushed a different solution

> ### 💪 3rd Party Providers for Fee & Exchange Rate

Author: https://github.com/WalletWasabi/WalletWasabi/pull/13738#discussion_r1945184993

Fee rate estimations and exchange rate providers are now configurable.

Previously, this information was provided by Wasabi's backend. This change aligns with our long-term goal of completely removing dependence on the backend.

> After this release, Wasabi will be significantly closer to achieving that goal, with most basic wallet features functioning without an active backend connection. You can read more about the remaining steps [here](https://github.com/orgs/WalletWasabi/discussions/13661).

Reviewer: nit
for the paranoid user,
it is not clear that clicking on the link will open the browser

### 👨‍🔧 Quality of Life Features

In the last version, we introduced a donation button to gauge community interest in funding Wasabi's future. We have been overwhelmed by the support and interest shown by our community, including long-time users and privacy enthusiasts.

We want to thank you for believing in Wasabi and for your support, whether through using the software, providing feedback, contributing financially, or contributing to the open-source project in any form.

To show our gratitude, we've designed this release for you, our community. We've finally addressed some of the oldest bugs and feature requests that have been long-awaited!

Listing all the improvements would be too lengthy, but here are some examples:
- Randomly skipping rounds was removed
- Coinjoin settings & profiles are now all in a single tab
- More information in the status icon
- Resync button now available in Tools

> Etc...

Reviewer: ![image](https://github.com/user-attachments/assets/88db0875-cf19-482f-bc1c-a77de5a8507b)

I don't think the **Etc** is desired like this?

Author: I´m missing the line break. Rider is very annoying when working with MD

> We expect these improvements to significantly enhance the overall experience of using Wasabi as your daily Bitcoin wallet. Take a tour or read release details on GitHub to see all of them!

Reviewer: ```suggestion
We expect these improvements to significantly enhance the overall experience of using Wasabi as your daily Bitcoin wallet. Take a tour or read the release details on GitHub to see all of them!
```

### 📦 Backend and Coordinator packaged for Linux

> The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). This update simplifies the deployment process for Linux users, making it straightforward to set up their backend servers and coordinators.

Reviewer: ```suggestion
The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). 
```
this is basically the same as next sentence?
maybe remove?

Reviewer 2: Idk, I like what @turbolay does with the highlights, a mean for communicating views, ideas, benefits, and even to say "thank you" and not just list a condensed summary of "What's new".

Author: Thanks @lontivero !

@MarnixCroes I removed first line, you are right that 2nd one conveys nicely the idea already
